roles/setup/tasks/retrieve_certs.yml

```diff
@@ -18,6 +18,7 @@
     mode: get
     dest: "{{ s3['local_dir'] }}/{{ object_name }}"
     overwrite: different
+  ignore_errors: yes
```

Comment:
Oh this is a dev thing that leaked out - I use mkcert to generate keys and drop them in group_vars/secrets/openstax.*
For all of {dev|qa|staging|prod} hosted on openstax.org, this probably _should_ error. Though I suppose if the keys are missing, we'd find out anyway when HTTPS doesn't work. Hmm.